package.json

```diff
@@ -17,11 +17,15 @@
   "dependencies": {
     "axios": "^0.27.2",
     "date-fns": "^2.29.1",
+    "formik": "^2.2.9",
```

Comment:
Just analyze and compare if we can use React Hook Form over Formik. 

It appears that React Hook Form has frequent updates over Formik which have last published one year ago.

found some useful blogs on the internet

React Hook Form isolates input components from the others and prevents re-render the form for a single input. It avoids this unnecessary re-rendering. So it is a great advantage in terms of performance compared to Formik which updates every change in every input field